include/seqan3/core/parallel/detail/spin_delay.hpp

```diff
@@ -72,6 +75,10 @@ class spin_delay
     {
         #if defined(__SSE2__)  // AMD and Intel
             _mm_pause();
+	#elif defined(__armel__) || defined(__ARMEL__) // armel, but broken
```

Comment:
Why is this not covered by the final else block? Or does __armel__ also define __arm__ but not the other way around?

Reply:
Dropped